tests/helpers.ts

```diff
@@ -11,12 +11,23 @@ export const redirectUri = "https://localhost:3000/auth/callback";
 export const scopes = ["transactions"];
 export const state = "25843739712322056";
 
-export const createClient = (opts = {}) => {
+type ClientOpts = {
```

Comment:
Instead of duplicating the type here, can we use the one from `lib/types.ts`?
We could make it optional by using `Partial<ClientOpts>`